prow/plugins/size/size.go

```diff
@@ -230,10 +229,18 @@ func isPRChanged(pe github.PullRequestEvent) bool {
 	}
 }
 
-func sizesOrDefault(sizes *plugins.Size) plugins.Size {
-	if sizes == nil {
-		return defaultSizes
+func defaultIfZero(value, defaultValue int) int {
```

Comment:
So this is where pointers would be useful -- do we want to support someone setting any of these fields to 0 and not getting defaulted? @cjwagner

Reply:
I can't think of a reason to specify a value of 0 so I'd say we don't care about supporting that in this case.

Reply:
So we keep it like that?

Reply:
Keeping it the way you have it SGTM